deps/llvm.mk

```diff
@@ -508,7 +508,7 @@ $(eval $(call LLVM_PATCH,llvm-D88630-clang-cmake))
 endif
 $(eval $(call LLVM_PATCH,llvm-invalid-addrspacecast-sink)) # Still being upstreamed as D92210
 $(eval $(call LLVM_PATCH,llvm-11-D93154-globalisel-as))
-$(eval $(call LLVM_PATCH,llvm-11-D94813-mergeicmps))
+$(eval $(call LLVM_PATCH,llvm-11-D94813-mergeicmps)) # remove for LLVM 14
 $(eval $(call LLVM_PROJ_PATCH,llvm-11-AArch64-FastIsel-bug))
 $(eval $(call LLVM_PATCH,llvm-12-D97435-AArch64-movaddrreg))
 $(eval $(call LLVM_PROJ_PATCH,llvm-11-D97571-AArch64-loh)) # remove for LLVM 13
```

Comment:
> # remove for LLVM 13

Missed this?

Reply:
You are looking at the patch list for LLVM 12